src/styles/blocks/header.scss

```diff
@@ -0,0 +1,68 @@
+header {
```

Comment:
Don't use tag names for styling (except html and body)
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/41291c21-e8dd-4227-85ad-41b4c7200d01)